app/entry.js

```diff
@@ -6,11 +6,56 @@ require('./scss/base.scss');
 
 // npm modules
 const angular = require('angular');
+const ngRoute = require('angular-route');
 
 // angular modules
-angular.module('demoApp', []);
+angular.module('brewBuddy', [ngRoute])
```

Comment:
Since your config as gotten very long, its a good idea to put this into its one node module, require it in and inject it into your angular brewBuddy module.
this can make it easier to read you entry point